exercises/practice/diffie-hellman/tests/diffie-hellman.rs

```diff
@@ -12,6 +12,14 @@ fn test_private_key_in_range_key() {
     }
 }
 
+
+#[test]
+fn test_private_key_is_prime() {
+    let primes: Vec<u64> = vec![1, 2, 3, 5, 7, 11, 13];
```

Comment:
```suggestion
    let primes: Vec<u64> = vec![2, 3, 5, 7, 11, 13];
```

Reply:
Good ideas, I changed the test.

With regard to json, how about this one?
```
    {
      "uuid": "c3fa6fed-d1b3-45f6-8245-be99d6e593a1",
      "description": "private key is a prime",
      "property": "privateKey",
      "input": {"p" :15},
      "expected": {
        "oneOf": [2, 3, 5, 7, 11, 13]
      }
    },
```

Reply:
Seems reasonable. Others may have opinions on whether it's okay to use the `privateKey` property but it's not currently being used right now, so it would at least not conflict with anything.